app/views/AnalyticalFramework/ListWidgetPreview/index.tsx

```diff
@@ -194,6 +196,42 @@ function ListWidgetPreview<N extends string, T>(props: Props<N, T>) {
             />
         );
     }
+    if (widget.type === 'matrix-1d') {
```

Comment:
Oh no, we are going to have a very nice conflict

Reply:
haha yes